### DISCUSSION TITLE
LDAPS not working, due to cert error?

### DISCUSSION THREAD

**Discussion by gijs007** [netbox-community/netbox-docker#829] [Getting Help] [answered]
I'm running into an issue when enabling LDAPS.

The LDAP login works fine when the following environmental variable is set:
LDAP_IGNORE_CERT_ERRORS="true"

However once I set it to false it fails with the following error:
Caught LDAPError while authenticating username: SERVER_DOWN({'result': -1, 'desc': "Can't contact LDAP server", 'ctrls': [], 'info': '(unknown error code)'})

I've mounted a BASE64 encoded certificate at the following path: /etc/ssl/certs/ca.domain.com.crt
The certificate is working to authenticate with the same LDAP server for a different tool we're using (Zabbix). 

I can only guess that the path of the certificate is somehow wrong, what would be the right path for Netbox?

**Comment by tobiasge** [comment, marked as answer]
I think placing the certificate in this path only works if you run `update-certificates` after.
Have a look at this comment:
https://github.com/netbox-community/netbox-docker/issues/799#issuecomment-1184251874

You can mount the certificate in a freely chosen place and then set the values mentioned.

**Reply by gijs007** [reply]
This was indeed the issue. Can I contribute by updating the wiki?
It was quite a challenge to find all the information to secure Netbox, to get it up and running for a production environment inside Docker. I'd gladly share my findings to make installing it a smoother process for all.
